src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -608,6 +611,27 @@ void ObjectMgr::LoadCreatureTemplateAddons()
             creatureAddon.emote = 0;
         }
 
+        if (!sAnimKitStore.LookupEntry(creatureAddon.aiAnimKit))
```

Comment:
`if (creatureAddon.aiAnimKit && !sAnimKitStore.LookupEntry(creatureAddon.aiAnimKit))`

Should only be checked if not 0 (and then the if inside this one is not required)